common/static/common/js/components/views/feedback.js

```diff
@@ -65,6 +67,39 @@
                     return this;
                 },
 
+                inFocus: function() {
+                    this.options.outFocusElm = this.options.outFocusElm || document.activeElement;
+
+                    // Set focus to the container.
+                    $(this.el).find(".wrapper").first().focus();
+
+                    /**
+                     * Make tabs within the prompt loop rather than setting focus
+                     * back to the main content of the page.
+                     */
+                    var tabbables = $(this.el).find(":tabbable");
+                    tabbables.on("keydown", function (event) {
+                        // On tab backward from the first tabbable item in the prompt
+                        if (event.which == 9 && event.shiftKey && event.target === tabbables.first()[0]) {
```

Comment:
Should === to avoid type coercion. @clytwynec make sure you have run the JavaScript linter on the code and fix issues it identifies. I think the linter runs now as part of the build process, but I don't know how to view it. The IDE I use (Pycharm) shows JS linting issues directly in the editor, so it is easy to fix them at the time you are writing the code.